lib/index.js

```diff
@@ -106,8 +106,10 @@ FacebookPixel.prototype.track = function(track) {
   }
 
   // standard conversion events, mapped to one of 9 standard events
+  // "Purchase" requires a currency parameter; default to USD
   // send full transformed payload
   each(function(event) {
+    if (event === 'Purchase') payload.currency = payload.currency || 'USD';
```

Comment:
We can use `.currency()` facade function which fallsback to `USD` albeit this is more explicit in what it does haha @sperand-io